### PR TITLE
Kafka : remove noise from logging EOF messages in Kafka parser

### LINKED ISSUE #3792: Kafka : remove noise from logging EOF messages in Kafka parser
We keep seeing a lot of these on normal client (produce/consume) connection close.
We should not be logging valid EOF as errors.

```
level=error msg="Unable to parse Kafka request; closing Kafka request connection" error=EOF id="rx:10.15.161.35:57590->10.15.28.238:10551<->tx:closed"
```

### PR DESCRIPTION
**Pull request by manalibhutiyani**
We keep seeing a lot of these on normal client (produce/consume) connection close.
We should not be logging valid EOF as errors.
```
level=error msg="Unable to parse Kafka request; closing Kafka request connection" error=EOF id="rx:10.15.161.35:57590->10.15.28.238:10551<->tx:closed"
```

Fixes: #3792
Signed-Off-By: Manali Bhutiyani <manali@covalent.io>

```release-note
Kafka : remove noise from logging EOF messages in Kafka parser
```

### CODE REVIEW

**Comment by manalibhutiyani** [comment]
test-me-please

**Comment by manalibhutiyani** [comment]
test-me-please

**Review by rlenglet — APPROVED** [review]

File: pkg/proxy/kafka.go [hunk 1]
```diff
@@ -31,6 +31,7 @@ import (
 
 	"github.com/optiopay/kafka/proto"
 	"github.com/sirupsen/logrus"
+	"io"
```
Comment (rlenglet) [inline pkg/proxy/kafka.go]: Move `"io"` and `"net"` into the stdlib group of imports above.

**Review by rlenglet — APPROVED** [review]

File: pkg/proxy/kafka.go (hunk 1 above)
Comment (manalibhutiyani) [reply pkg/proxy/kafka.go]: will do.

Comment (manalibhutiyani) [reply pkg/proxy/kafka.go]: done

**Comment by manalibhutiyani** [comment]
test-me-please

**Review by tgraf — APPROVED** [review]

**Comment by raybejjani** [comment]
backport https://github.com/cilium/cilium/pull/3811

**Pull request merged**